boards/norik/octopus_som/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Norik Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config OCTOPUS_SOM_CONTROL_INIT_PRIORITY
+	int "Init priority"
+	default 99
```

Comment:
`select PINCTRL`

Reply:
Because you have a .c file which is using it. Breaks how? If it is a Kconfig error then add a new hidden Kconfig that selects that symbol

Reply:
I've added `config BOARD_OCTOPUS_SOM` that contains `select PINCTRL`